apps/microsoft-teams/app-actions/src/helpers/entry-activity.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { buildEntryActivity } from './entry-activity';
+import {
+  CollectionProp,
+  ContentTypeProps,
+  LocaleProps,
+  PlainClientAPI,
+  UserProps,
+} from 'contentful-management';
+import {
+  makeMockPlainClient,
+  mockContentType,
+  mockEntryEvent,
+  mockLocaleCollection,
+  mockUser,
+} from '../../test/mocks';
+import { EntryEvent } from '../types';
+
+// TDOO Could add some more tests around user activity (choosing which user did the thing)
+describe('buildEntryActivity', () => {
+  let cmaRequestStub: sinon.SinonStub;
+  let cmaClientMockResponses: [ContentTypeProps, CollectionProp<LocaleProps>, UserProps];
+  let cma: PlainClientAPI;
+  let entryEvent: EntryEvent;
+
+  beforeEach(() => {
+    cmaRequestStub = sinon.stub();
+    cmaClientMockResponses = [mockContentType, mockLocaleCollection, mockUser];
+    cma = makeMockPlainClient(cmaClientMockResponses, cmaRequestStub);
+    entryEvent = { ...mockEntryEvent };
+  });
+
+  it('returns an entry activity object', async () => {
+    const result = await buildEntryActivity(entryEvent, cma);
+    expect(result).to.have.property('spaceName', 'TODO: Space name');
```

Comment:
We should remove `spaceName` and `actorName` here or in a subsequent PR

Reply:
added TODO